CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
 # Changelog
 
-## [unreleased]
+## unreleased
+
+### Protocol Changes
+
+*
+
+### Non-protocol Changes
+
+* `neard` now accepts `--home` and `--verbose` anywhere in the command line: `neard run --home ./near`.
+
+
+## 1.29.0 []
```

Comment:
What happened with our changelog? I though we've branched 1.30 already, no?

Reply:
Nobody understands how CHANGELOG is supposed to work.  cc @marcelo-gonzalez  

By the way, this is part of the reason I keep suggesting mandator RELNOTES in commit message enforced by a precommit.